app/views/posts/_article_list.html.erb

```diff
@@ -21,13 +21,30 @@
       <% end %>
       <%= link_to post.title, generic_share_link(post), 'data-ckb-item-link' => '' %>
     </div>
-    <% if @last_activity %>
-      <p class="has-color-tertiary-600 has-float-right post-list--meta">
+    <p class="has-color-tertiary-600 has-float-right post-list--meta"> posted <span title="<%= post.created_at.iso8601 %>"><%= time_ago_in_words(post.created_at, locale: :en_abbrev) %> ago</span>
+      by <%= link_to post.user.rtl_safe_username, user_path(post.user), dir: 'ltr' %>&nbsp;&middot;&nbsp;
+      <% if post.last_activity == post.closed_at %>
+        Closed
+        <span title="<%= post.last_activity.iso8601 %>"><%= time_ago_in_words(post.last_activity, locale: :en_abbrev) %> ago</span>
+        by <%= link_to active_user.rtl_safe_username, user_path(active_user), dir: 'ltr' %>
+      <% elsif post.last_activity == post.locked_at %>
+        Locked
+        <span title="<%= post.last_activity.iso8601 %>"><%= time_ago_in_words(post.last_activity, locale: :en_abbrev) %> ago</span>
+        by <%= link_to active_user.rtl_safe_username, user_path(active_user), dir: 'ltr' %>
+      <% elsif post.last_activity == post.deleted_at %>
+        Deleted
+        <span title="<%= post.last_activity.iso8601 %>"><%= time_ago_in_words(post.last_activity, locale: :en_abbrev) %> ago</span>
+        by <%= link_to active_user.rtl_safe_username, user_path(active_user), dir: 'ltr' %>
+      <% elsif post.last_activity == post.last_edited_at %>
+        Edited
+        <span title="<%= post.last_activity.iso8601 %>"><%= time_ago_in_words(post.last_activity, locale: :en_abbrev) %> ago</span>
+        by <%= link_to active_user.rtl_safe_username, user_path(active_user), dir: 'ltr' %>
+      <% elsif post.last_activity != post.created_at %>
         last activity
-        <span title="<%= post.last_activity.iso8601 %>"><%= time_ago_in_words(post.last_activity) %> ago</span>
-        by <%= link_to active_user.rtl_safe_username, user_path(active_user), dir: 'ltr'%>
+        <span title="<%= post.last_activity.iso8601 %>"><%= time_ago_in_words(post.last_activity, locale: :en_abbrev) %> ago</span>
+        by <%= link_to active_user.rtl_safe_username, user_path(active_user), dir: 'ltr' %>
+      <% end %>
```

Comment:
Shouldn't first letter be capital ? e.g. `Closed`

Reply:
@ArtOfCode- OK. Just a little notice, By the `Post` Class, so you mean `/app/models/post.rb` file, no?

Reply:
> Shouldn't first letter be capital ? e.g. Closed

@Istiakshovon No, because that whole line is lowercase; a capital would be out of place there.

> @ArtOfCode- OK. Just a little notice, By the Post Class, so you mean /app/models/post.rb file, no?

@kevinmmansour Yep, that's right.

Reply:
@ArtOfCode- Done! I have updated `Post` Class and Articles View.